conf/defaults.ini

```diff
@@ -377,6 +377,15 @@ tls_client_key =
 tls_client_ca =
 send_client_credentials_via_post = false
 
+#################################### SAML Auth ###########################
+[auth.saml] # Enterprise only
+enabled = false
+private_key =
+certificate =
+idp_metadata =
```

Comment:
_path and _url versions need to be in here as well. If they are not here it will not be possible to configure the variables via the environment.

Reply:
done.